client/src/components/Login.vue

```diff
@@ -52,6 +68,8 @@ export default {
         }
       }
       console.log(formValues)
+      // restFactory.login((res) => {})
```

Comment:
will these commented out calls be of use in the immediate future? If not we should remove them.